frame/dynamic-fee/src/lib.rs

```diff
@@ -26,6 +26,8 @@ use sp_core::U256;
 use sp_inherents::{InherentData, InherentIdentifier};
 use sp_std::cmp::{max, min};
 
+use fp_dynamic_fee::{InherentType, INHERENT_IDENTIFIER};
```

Comment:
Can we add a pub reexport of those types (as well as `InherentDataProvider` in std) here?

Reply:
Done